packages/thrift-integration/tsconfig.json

```diff
@@ -1,22 +1,22 @@
 {
     "compilerOptions": {
-      "target": "es2017",
-      "module": "commonjs",
-      "moduleResolution": "node",
-      "sourceMap": true,
-      "declaration": true,
-      "rootDir": "./src",
-      "outDir": "./dist",
-      "noEmitOnError": true,
-      "strict": true,
-      "noUnusedLocals": true,
-      "pretty": true,
-      "removeComments": true
+        "target": "es2017",
```

Comment:
all the other `tsconfig.json` files have `"lib": ["es2017"]` before the `target` ... is the fact that it's not here significant?

Reply:
Yeah, it’s significant. By not using “lib” typescript includes a default set of type definitions for standard JS objects/functions. This includes things that are browser-specific. The integration tests need these because we test against Apache thrift that uses these types.

Reply:
👍